### DISCUSSION TITLE
continue on failure?

### DISCUSSION THREAD

**Discussion by gingerlime** [nginx-proxy/acme-companion#936] [Q&A] [answered]
This is more of a question than a bug, but I couldn't find another option.

I noticed that if the `LETSENCRYPT_HOST` contains multiple hosts, and one of them fails, then it can cause the others to fail as well ... I wonder if there's a way to skip any failed host names, but still produce a valid certificate for the valid hostnames?

**Comment by buchdag** [comment, marked as answer]
Hi. I think you could use the [separate certificate feature](https://github.com/nginx-proxy/acme-companion/blob/main/docs/Let's-Encrypt-and-ACME.md#separate-certificate-for-each-domain) for this. The domains in `LETSENCRYPT_HOST` will each be issued a separate certificate, and each certificate will be verified separately.

**Comment by gingerlime** [comment]
Thank you. I missed it. So using `LETSENCRYPT_SINGLE_DOMAIN_CERTS=true` would issue separate certificates for each domain, rather than try to combine them into one certificate with all domains, and therefore can bypass failures, right?
